draft-ietf-tls-dtls-connection-id.md

```diff
@@ -154,32 +154,24 @@ DTLS 1.2 in general does not allow TLS 1.3-style post-handshake messages
 that do not themselves begin other handshakes. When a DTLS session is
 resumed or renegotiated, the "connection_id" extension is negotiated afresh.
 
-If DTLS peers have not negotiated the use of CIDs, which includes the case
-where both sent a zero-length cid in their connection_id extensions, then the
-RFC 6347-defined record format and content type MUST be used.
-
-If DTLS peers have negotiated the use of a non-zero-length CID in at least one
-direction using the ClientHello and the ServerHello messages, then the peers
-need to take the following steps.
-
-The DTLS peers determine whether incoming or outgoing (or both) messages need
-to use the new record format, i.e., the record format containing the CID.
-The new record format with the the tls12_cid content type is only used once encryption
-is enabled. Plaintext payloads never use the new record type and the CID content
+If DTLS peers have not negotiated the use of CIDs, or a zero-length
+CID has been advertised for a given direction, then the RFC
+6347-defined record format and content type MUST be used to send in
+that direction.
```

Comment:
(nit) something of a singular/plural mismatch since "not negotiated the use of CIDs" doesn't have an associated direction.
On first look, "MUST be used to send in the indicated direction(s)" might cover it.